docs/supported_apis/dataframe_supported.rst

```diff
@@ -299,23 +320,23 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``resample``               | `resample`_               | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``reset_index``            | `reset_index`_            | Y                      |                                                    |
+| ``reset_index``            | `reset_index`_            | Y                      | **Omnisci**:``P``, ``D`` for ``level`` parameter   |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``rfloordiv``              | `rfloordiv`_              | Y                      | See ``add``                                        |
+| ``rfloordiv``              | `rfloordiv`_              | Y                      | See ``add``; **Omnisci**:``N``                     |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``rmod``                   | `rmod`_                   | Y                      | See ``add``                                        |
+| ``rmod``                   | `rmod`_                   | Y                      | See ``add``; **Omnisci**:``N``                     |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``rmul``                   | `rmul`_                   | Y                      | See ``add``                                        |
+| ``rmul``                   | `rmul`_                   | Y                      | See ``add``; **Omnisci**:``N``                     |
```

Comment:
`rmul` should be supported as well as `mul`, at least I get no warnings for it:
```python
>>> df.rmul(2)
   a   b
0  2   6
1  4   8
2  6  10
3  8  12
```

List of supported binary operations by Omnisci [[1]](https://github.com/modin-project/modin/blob/5440f08c1dca02f12ea4a4479e6d3ea91cef5a66/modin/experimental/core/storage_formats/omnisci/query_compiler.py#L621-L661)

Reply:
Fixed